### PR TITLE
Fix KEYCLOAK_EXTRA_ARGS enviroment on run script (issue #58)

### PR DESCRIPTION
**Pull request by mancausoft**
Change script run.sh to really use the enviroment variable KEYCLOAK_EXTRA_ARGS
See issue #58

### CODE REVIEW

File: 17/debian-10/rootfs/opt/bitnami/scripts/keycloak/run.sh [hunk 1]
```diff
@@ -26,7 +26,7 @@ start_command=("${KEYCLOAK_BIN_DIR}/kc.sh -cf ${conf_file} ${start_param}")
 # Add extra args
 if [[ -n "$KEYCLOAK_EXTRA_ARGS" ]]; then
     read -r -a extra_args <<<"$KEYCLOAK_EXTRA_ARGS"
-    start_command+=("${extra_args[@]}")
+    start_command+=" ${extra_args[@]}"
```
Comment (joancafom) [inline 17/debian-10/rootfs/opt/bitnami/scripts/keycloak/run.sh:29]: I don't think the real issue comes from here, but from the way the variables are passed when the command is executed. The `a+=( "${b[@]}")` syntax should be totally fine when working with arrays :)

Comment (joancafom) [reply 17/debian-10/rootfs/opt/bitnami/scripts/keycloak/run.sh:29]: The error IMO comes from these two lines:

1. Each component should be an individual array in `start_command`:

https://github.com/bitnami/bitnami-docker-keycloak/blob/77ffc2ea48a061933aafe24fc086a0d9e480ba38/18/debian-10/rootfs/opt/bitnami/scripts/keycloak/run.sh#L24

```diff
- start_command=("${KEYCLOAK_BIN_DIR}/kc.sh -cf ${conf_file} ${start_param}")
+ start_command=("${KEYCLOAK_BIN_DIR}/kc.sh" "-cf" "$conf_file" "$start_param")
```

2. The array should be expanded as **a single string**
https://github.com/bitnami/bitnami-docker-keycloak/blob/77ffc2ea48a061933aafe24fc086a0d9e480ba38/18/debian-10/rootfs/opt/bitnami/scripts/keycloak/run.sh#L32-L36

```diff
if am_i_root; then
-   exec gosu "$KEYCLOAK_DAEMON_USER" /bin/bash -c "${start_command[@]}"
+   exec gosu "$KEYCLOAK_DAEMON_USER" /bin/bash -c "${start_command[*]}"
else
-   exec /bin/bash -c "${start_command[@]}"
+   exec /bin/bash -c "${start_command[*]}"
fi
```

**Review by joancafom — CHANGES_REQUESTED** [review]
Thanks for your PR! I left you some comments and a recommended approach, can you check it and verify it works as expected?

File: 17/debian-10/rootfs/opt/bitnami/scripts/keycloak/run.sh (hunk 1 above)
Comment (mancausoft) [reply 17/debian-10/rootfs/opt/bitnami/scripts/keycloak/run.sh:29]: Yes your solution is more clean. I tested and seems working
```
++ exec /bin/bash -c '/opt/bitnami/keycloak/bin/kc.sh -cf /opt/bitnami/keycloak/conf/keycloak.conf start-dev -Dkeycloak.profile=preview -Dkeycloak.profile.feature.token_exchange=enabled -Dkeycloak.profile.feature.admin_fine_grained_authz=enabled -Dkeycloak.profile.feature.script=enable'
Updating the configuration and installing your custom providers, if any. Please wait.
2022-05-24 08:13:15,388 INFO  [org.keycloak.common.Profile] (build-47) Preview feature enabled: admin_fine_grained_authz
2022-05-24 08:13:15,422 INFO  [org.keycloak.common.Profile] (build-47) Preview feature enabled: admin2
2022-05-24 08:13:15,423 INFO  [org.keycloak.common.Profile] (build-47) Preview feature enabled: openshift_integration
2022-05-24 08:13:15,424 INFO  [org.keycloak.common.Profile] (build-47) Preview feature enabled: scripts
2022-05-24 08:13:15,424 INFO  [org.keycloak.common.Profile] (build-47) Preview feature enabled: token_exchange
2022-05-24 08:13:15,426 INFO  [org.keycloak.common.Profile] (build-47) Preview feature enabled: declarative_user_profile
2022-05-24 08:13:15,426 INFO  [org.keycloak.common.Profile] (build-47) Preview feature enabled: client_secret_rotation
2022-05-24 08:13:15,427 INFO  [org.keycloak.common.Profile] (build-47) Preview feature enabled: recovery_codes```

i'm pushing the change

**Review by joancafom — APPROVED** [review]
LGTM! Thank you so much for your PR 🚀

**Pull request merged**